src/server/v2.0/handler/ldap.go

```diff
@@ -0,0 +1,109 @@
+package handler
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/goharbor/harbor/src/common/rbac"
+	"github.com/goharbor/harbor/src/common/security"
+	ctlLdap "github.com/goharbor/harbor/src/controller/ldap"
+	"github.com/goharbor/harbor/src/lib/errors"
+	ldapModel "github.com/goharbor/harbor/src/pkg/ldap/model"
+	"github.com/goharbor/harbor/src/server/v2.0/models"
+	"github.com/goharbor/harbor/src/server/v2.0/restapi/operations/ldap"
+)
+
+type ldapAPI struct {
+	BaseAPI
+	ctr ctlLdap.Controller
```

Comment:
`ctl ctlLdap.Controller` may be better than `ctr ctlLdap.Controller` because in other handlers we always use `ctl` or `xxxCtl`.

And also we can alias `github.com/goharbor/harbor/src/server/v2.0/restapi/operations/ldap` to `operation` so that we can use `ctl ldap.Controller` here.

Reply:
Updated